source/features/highest-rated-comment.css

```diff
@@ -1,5 +1,17 @@
+.rgh-highest-rated-comment {
+	border: 2px solid var(--color-text-success);
+}
+
+.rgh-highest-rated-comment.timeline-comment--caret::before {
+	background-color: var(--color-text-success);
+}
+
+.rgh-highest-rated-comment.timeline-comment--caret::after {
+	left: -6px;
```

Comment:
Can you leave a comment describing what this does? That is: "It makes the caret’s border 2px"

Reply:
Resolved in 108aa74